apps/jobs/views.py

```diff
@@ -322,6 +322,47 @@ def challenge_submission(request, challenge_id, challenge_phase_id):
         )
 
 
+@api_view(["DELETE"])
+@throttle_classes([UserRateThrottle])
+@permission_classes((permissions.IsAuthenticated, HasVerifiedEmail))
+@authentication_classes((ExpiringTokenAuthentication,))
+def delete_challenge_submission(request, challenge_pk, challenge_phase_pk, submission_pk):
```

Comment:
Hey @Kajol-Kumari Do we really need challenge phase id here when we already have submission id

Reply:
@Sanji515 Incorporated the changes. Please take a look it.